src/test_workflow/integ_test/integ_test_suite_opensearch_dashboards.py

```diff
@@ -58,9 +59,13 @@ def __init__(
 
         # Integ-tests for OSD now clones FunctionalTestDashboards Repository by default and points to integtest.sh from FunctionalTestDashboards for all OSD plugins
 
+        functionalTestDashboards_ref = ft_repo_ref
+        if ft_repo_ref is not None:
```

Comment:
Shouldn't this be `is None`? If the user has not provided the ref it should then pick up from build manifest right?

Reply:
Yep, agreed.